tinker/static/assets/css/tinker.css

```diff
@@ -175,6 +180,11 @@ a#remove_new_job_button {
     border-color: #f04124 !important;
 }
 
+.btn-danger {
+    background-color: #f04124 !important;
+    border-color: #f04124 !important;
+}
```

Comment:
Do we need a custom `btn-danger` class? or can we get by with the default? https://getbootstrap.com/docs/4.3/components/buttons/

Reply:
Sort of.

With the custom btn-danger class our button looks like this:
![image](https://user-images.githubusercontent.com/26125143/59270736-de1ba180-8c17-11e9-8fcf-a6c102e673a6.png)

Without the class the button looks like this:
![image](https://user-images.githubusercontent.com/26125143/59270758-ebd12700-8c17-11e9-80ac-43b34f81ca67.png)

And without any of the CSS that changes the button blue, it looks like this, even though the btn-danger class is applied to the button and the btn-outline-danger class is not:
![image](https://user-images.githubusercontent.com/26125143/59270776-f7bce900-8c17-11e9-86ff-a7e57b58e89e.png)

Reply:
Gotcha. Lets go ahead and push this through